src/main/java/org/mskcc/limsrest/service/ilabs/CustomForm.java

```diff
@@ -0,0 +1,53 @@
+package org.mskcc.limsrest.service.ilabs;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CustomForm {
+    private String id;
+    private String name;
+    private String note;
+    private Map<String, String> fields;
+
+    public CustomForm() {
```

Comment:
For pulling iLab to Teamwork cards I've already added this class under [src/main/java/org/mskcc/limsrest/service/]. So, this is redundant.

Reply:
ok